src/redux/categories/categories.js

```diff
@@ -0,0 +1,28 @@
+// Action types
+const CHECK_STATUS = 'check status';
```

Comment:
- [ ] Hi. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 
 i.e  `const CHECK_STATUS = redux/books/CHECK_STATUS `(following your workflow)